components/chainhook-sdk/src/serde_hex.rs

```diff
@@ -0,0 +1,13 @@
+
+use bitcoin::hex::{DisplayHex, FromHex};
+use serde::de::Error;
+use serde::{Deserializer, Serializer};
+
+pub fn serialize<S: Serializer>(b: &Vec<u8>, s: S) -> Result<S::Ok, S::Error> {
```

Comment:
Do we still need this? Maybe with all the code we deleted it's no longer necessary. Could you double check?